### ISSUE TITLE
Clustering: algorithm and related library uses (sci-kit learn)

### ISSUE THREAD

**Issue by sophryu99** [sophryu99/TIL#6]
## Clustering with scikit-learn

`sklearn.cluster`
Clustering algorithms in scikit-learn: [documentation](https://scikit-learn.org/stable/modules/clustering.html)
<img width="700" alt="Screen Shot 2021-07-28 at 2 36 53 PM" src="https://user-images.githubusercontent.com/46921003/127269342-699e48f8-61f7-48ee-a8fa-7355f1e798b5.png">

**Comment by sophryu99** [comment]
### **Clustering vs Classification**

**Clustering**

- Ideally, captures generating distributions

- Practically, is an exploration of the structure of your dataset

- **Data points are fixed**
- Cluster centers are searched for



**Classification**

- Labels are fixed

- Transformations of the data points are searched for
